tagstudio/src/core/library/alchemy/library.py

```diff
@@ -957,13 +937,17 @@ def update_subtags(self, tag, subtag_ids, session):
     def prefs(self, key: LibraryPrefs) -> Any:
         # load given item from Preferences table
         with Session(self.engine) as session:
-            return session.scalar(select(Preferences).where(Preferences.key == key.name)).value
+            pref = session.scalar(select(Preferences).where(Preferences.key == key.name))
+            assert pref is not None
```

Comment:
Any reason for not raising a KeyError like 10 lines down?

Reply:
There’s no specific reason. i didn't notice that. i guess, in most cases this won’t return `None`. so these are mainly to satisfy mypy.